app/routes/__index/certificate/index.tsx

```diff
@@ -35,19 +35,29 @@ export default function CertificateIndexRoute() {
     }, 5000);
   }
 
+  function dateFormat(val: Date): string {
+    let date = val.toLocaleDateString('en-US', {
```

Comment:
value of the `date` is not immutable so no need to declare it as `let` + you could just write it simpler:
```typescript
function dateFormat(val: Date): string {
return val.toLocaleDateString('en-US', {
month: 'short',
      day: '2-digit',
      year: 'numeric',
    });
}
```

I also think that the function name should be something like `formatDate` or `getFormatedDate`, `dateFormat` sounds like a variable name to me

Reply:
yeah you're right, I like `formatDate` better